plenum/common/ledger_manager.py

```diff
@@ -86,6 +88,29 @@ def addLedger(self, iD: int, ledger: Ledger,
             verifier=MerkleVerifier(ledger.hasher)
         )
 
+    def _cancel_request_ledger_statuses_and_consistency_proofs(self, ledger_id):
+        if ledger_id in self.request_ledger_status_action_ids:
+            self._cancel(self.request_ledger_status_action_ids[ledger_id])
+            self.request_ledger_status_action_ids[ledger_id] = None
```

Comment:
The way to remove a cancelled action from the map and the the condition being verified above mismatch. It will be uniform and simpler to remove an entry from the map rather than to use `None` value.

Reply:
Do you mean something like this?
```
if ledger_id in self.request_ledger_status_action_ids:
            action = self.request_ledger_status_action_ids.pop(ledger_id)
            self._cancel(action)
```

Reply:
Done.